ably.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,9 @@
 <Workspace
    version = "1.0">
    <FileRef
+      location = "group:ablySpec/ReadmeExamples.swift">
```

Comment:
I always _gitignore_ this file. Never notice it was here.
Can you remove it and add it to the `.gitignore`?
This file is generated by the CocoaPods.